routes/api/v1/dashboards.js

```diff
@@ -0,0 +1,69 @@
+const Dashboard = require('../../../models/dashboards.js')
+const { Widget } = require('../../../models/widgets.js')
+
+const express = require('express')
+const router = express.Router()
+
+router.get('/', (request, response, next) => {
+  Dashboard.find().exec()
+    .then( dashboard => response.json( dashboard ) )
+    .catch( error => next( error ) )
+})
+
+router.post('/', (request, response, next) => {
+  Dashboard.create( request.body )
+    .then( dashboard => response.json( dashboard ) )
+    .catch( error => next( error ) )
+})
+
+router.get('/:id', (request, response, next) => {
+  Dashboard.findById( request.params.id ).exec()
+    .then( dashboard => response.json( dashboard ) )
+    .catch( error => next( error ) )
+})
+
+router.put('/:id', (request, response, next) => {
+  Dashboard.findByIdAndUpdate( request.params.id, request.body ).exec()
```

Comment:
I am curious to know what happens when we do something like use Postman to issue a `PUT` request to this endpoint for a valid id, but with some random data:
```
{ blarg: 'thing', saxifrage: 42 }
```
Does that information get stored in this record?